addon/components/frost-textarea.js

```diff
@@ -49,58 +65,71 @@ export default Component.extend(PropTypeMixin, {
     }
   },
 
-  // == Computed Properties ===================================================
-
-  @readOnly
-  @computed('disabled', 'readonly', 'value')
-  /**
-   * Determine whether or not to show button for clearing out text field
-   * @param {Boolean} disabled - whether or not input is disabled
-   * @param {Boolean} readonly - whether or not input is readonly
-   * @param {String} value - value of text field
-   * @returns {Boolean} whether or not to show button for clearing out text field
-   */
-  showClear (disabled, readonly, value) {
-    return !disabled && !isEmpty(value) && !readonly
+  // // == Events ================================================================
+
+  init () {
+    this._super(...arguments)
+    this.receivedHook = this.hook
+    if (get(this, 'isHookEmbedded')) {
+      this.hook = ''
+    }
```

Comment:
You will not need to have this `init()` block of code. It was in `frost-text` to keep track of hooks since `frost-text` was embedded in `frost-password`.